doc/releases/release-notes-template.rst

```diff
@@ -0,0 +1,269 @@
+:orphan:
+
+.. _zephyr_X.Y:
+
+Zephyr X.Y.0 (Working Draft)
+############################
+
+We are pleased to announce the release of Zephyr version X.Y.0.
+
+.. This section is to be populated by the release manager
+
+Major features and improvements in this release include:
+
+*
+*
+
+API Changes
+***********
+
+.. Please document all API additions, removals, deprecation and changes here and
+   not in the individual subsections
+
+New APIs
+========
+
+Removed APIs
+============
+
+Deprecated APIs
+===============
+
+Changed APIs
+============
+
+.. Includes both stable and unstable/experimental API changes
+
+Security Vulnerabilities
+************************
+
+Detailed changelog
+******************
+
+Kernel
+======
+
+Architectures
+=============
+
+* ARC
+
+* ARM
+
+* ARM64
+
+* MIPS
+
+* NIOS2
+
+* POSIX
+
+* RISC-V
+
+* SPARC
+
+* x86
+
+* Xtensa
+
+Bluetooth
+=========
+
+* Audio
+
+* Direction Finding
+
+* Host
+
+* Mesh
+
+* Controller
+
+* HCI Driver
+
+Boards & SoC Support
+====================
+
+* SoC series
+
+* ARC boards
+
+* ARM boards
+
+* ARM64 boards
+
+* MIPS boards
+
+* NIOS2 boards
+
+* POSIX boards
+
+* RISC-V boards
+
+* SPARC boards
+
+* x86 boards
+
+* Xtensa boards
+
+* Other boards
+
+* Shields:
+
+Build and configuration systems
+===============================
+
+Devicetree
+==========
+
+Device Management
+=================
+
+Documentation
+=============
+
+Drivers
+=======
+
+* ADC
+
+* Battery-backed RAM
+
+* CAN
+
+* Clock control
+
+* Counter
+
+* Crypto
+
+* DAC
+
+* DFU
+
+* Disk
+
+* Display
+
+* DMA
+
+* EEPROM
+
+* Entropy
+
+* ESPI
+
+* Ethernet
+
+* Flash
+
+* FPGA
+
+* Fuel Gauge
+
+* GPIO
+
+* hwinfo
+
+* I2C
+
+* I2S
+
+* I3C
+
+* IEEE 802.15.4
+
+* Interrupt Controller
+
+* IPM
+
+* KSCAN
+
+* LED
+
+* MBOX
+
+* MEMC
+
+* PCIE
+
+* PECI
+
+* Pin control
+
+* PWM
+
+* Power domain
+
+* Regulators
+
+* Reset
+
+* RTC
+
+* SDHC
+
+* Serial
+
+* SPI
+
+* Timer
+
+* USB
+
+* W1
+
+* Watchdog
+
+* WiFi
+
+Logging
+=======
+
+Modules
+=======
+
+HALs
+----
+
+mbedTLS
+-------
+
+MCUboot
+-------
+
+Trusted Firmware-M
+------------------
+
+zcbor
+-----
+
+Networking
+==========
+
+Other Subsystems and Libraries
+==============================
+
+Sensors
```

Comment:
Sensors belongs under drivers.

Reply:
I decided to move it out because I found it very confusing, and found myself looking for the heading multiple times.

Reply:
Sensors will soon have their own subsystem "Sensing" https://github.com/zephyrproject-rtos/zephyr/pull/55389

So moving forward, I think sensors belong here.

Reply:
> I decided to move it out because I found it very confusing, and found myself looking for the heading multiple times.

The heading will still be there, just one level lower - same level as all other driver class changes.